pkgs/development/tools/bingo/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, buildGoModule, fetchFromGitHub, testVersion, bingo }:
+
+buildGoModule rec {
+  pname = "bingo";
+  version = "0.5.2";
+
+  src = fetchFromGitHub {
+    owner = "bwplotka";
+    repo = "bingo";
+    rev = "v${version}";
+    sha256 = "sha256-4D8YaA/AH1gIp5iwD7WEAdBl73sqwHpfOe7bnxVcRcw=";
+  };
+
+  vendorSha256 = "sha256-xrz9FpwZd+FboVVTWSqGHRguGwrwE9cSFEEtulzbfDQ=";
+
+  patches = [
+    # Skip e2e tests.
+    ./e2e_test.patch
```

Comment:
Probably easier to set `doCheck = false` than keeping a 1300 line patch around.

Reply:
or do `rm` on the deleted file.

Reply:
Thanks guys, this patch deletes two test files. `doCheck = false` sounds good, But I do not want to skip the check phase.

Reply:
Don't need to patch them out, just delete them.

```nix
  postPatch = ''
    rm get_e2e_test.go get_e2e_utils_test.go
  '';
```

Reply:
I see. Nice.